homeassistant/components/airzone/const.py

```diff
@@ -0,0 +1,43 @@
+"""Constants for the Airzone integration."""
+
+from typing import Final
+
+from aioairzone.const import AZD_HUMIDITY, AZD_TEMP
+
+from homeassistant.components.sensor import (
+    STATE_CLASS_MEASUREMENT,
+    SensorEntityDescription,
+)
+from homeassistant.const import (
+    DEVICE_CLASS_HUMIDITY,
+    DEVICE_CLASS_TEMPERATURE,
+    ENTITY_CATEGORY_DIAGNOSTIC,
+    PERCENTAGE,
+    TEMP_CELSIUS,
+)
+
+DOMAIN = "airzone"
```

Comment:
```suggestion
DOMAIN: Final = "airzone"
```

Reply:
Changed in 94785a97371d4cca3bf7b1ec50f32a046a8488f2